sklearn/linear_model/passive_aggressive.py

```diff
@@ -282,8 +282,8 @@ class PassiveAggressiveRegressor(BaseSGDRegressor):
     early_stopping : bool, default=False
         Whether to use early stopping to terminate training when validation.
         score is not improving. If set to True, it will automatically set aside
-        a fraction of training data as validation and terminate training when
-        validation score is not improving by at least tol for
+        a fraction of training data as validation and terminate
```

Comment:
You forgot to add "stratified" here and there.

Reply:
These are regressors

Reply:
Right, my mistake, I was fooled by the changes.